plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveStorageFormat.java

```diff
@@ -210,4 +210,12 @@ public String humanName()
             default -> toString();
         };
     }
+
```

Comment:
I believe you copied code from #22177. Please add `Co-authored-by` with the original author to the commit.